index.html

```diff
@@ -45,6 +45,106 @@ <h1>
 
         <a href="#" class="arrow"><img src="images/Disabled.png" alt="" /></a>
       </section>
+
+      <section class="my-works">
+        <h2>My Recent Works</h2>
+        <img src="images/Indicator.png" alt="indicator" />
+        <div class="Works">
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
```

Comment:
- [ ] Instead of using `div & span` tag here. Let's use the `ul & li` tag here. Kindly apply the change to all the places in the `works section`.
- **Requirement**  The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`